docs/source/addons/i18n.md

```diff
@@ -28,7 +28,7 @@ Your add-on has a `locales` folder with a `.pot` file.
 
 In the context of your project, run `yarn i18n` to merge the add-on translations with the ones of your project.
 
-
```

Comment:
Please do not remove deliberate white space. It makes it easier for authors to quickly find sections. Revert.